libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/CausalIndividualChart.tsx

```diff
@@ -458,6 +458,40 @@ export class CausalIndividualChart extends React.PureComponent<
         trace.y = rawY;
       }
     }
+    if (
+      this.context.jointDataset.datasetMetaData?.featureMetaData !== undefined
```

Comment:
maybe use:
```
!this.context.jointDataset.datasetMetaData?.featureMetaData
```
instead of comparing to undefined

Reply:
Done